pycaw/pycaw.py

```diff
@@ -709,8 +709,13 @@ def CreateDevice(dev):
             propCount = store.GetCount()
             for j in range(propCount):
                 pk = store.GetAt(j)
-                value = store.GetValue(pk)
-                v = value.GetValue()
+                try:
+                    value = store.GetValue(pk)
+                    v = value.GetValue()
+                except:
```

Comment:
Let's be more specific with a `except _ctypes.COMError:`
Also I would print out a warning in the console to log the error. Sometimes failing too gracefully is a nightmare to debug later. So maybe something like.
```
except _ctypes.COMError as e:
    warnings.warn("COMError", e)
```
This case would also deserve some unit tests I think